$OpenBSD: patch-dialects_n+obsd_machine_h,v 1.2 2013/05/23 15:08:36 dcoppa Exp $

XXX lsof needs adapting to cope with changes made to the name cache
at f2k9 (Aug 2009). for now, disable name cache support so it works.

--- dialects/n+obsd/machine.h.orig	Thu Jul 29 18:02:55 2010
+++ dialects/n+obsd/machine.h	Wed May 22 10:25:32 2013
@@ -262,7 +262,7 @@
  * NCACHELDSFX is a set of C commands to execute after calling ncache_load().
  */
 
-#define	HASNCACHE	1
+/* #define	HASNCACHE	1 */
 /* #define	NCACHELDPFX	??? */
 /* #define	NCACHELDSFX	??? */
 
